tests/network/l2_bridge/test_l2_ovs_linux_bridge.py

```diff
@@ -69,9 +69,9 @@ def test_connectivity_l2_bridge(
     @pytest.mark.polarion("CNV-2282")
     def test_dhcp_broadcast(
         self,
-        configured_l2_bridge_vm_a,
-        l2_bridge_running_vm_b,
         dhcp_nad,
+        l2_bridge_running_vm_b,
+        configured_l2_bridge_vm_a,
```

Comment:
What are these changes for? If it was because you wanted the NAD to be created first, it is wrong to depend on the order of fixture listing and in this case not needed (because each fixture already depends on the NAD).

Reply:
It is incorrect and bad practice to depend on the definition order of fixtures in the tests.
If it was not a must, this change should not have been done at all.

There may be exceptional cases when there is no other choice but to depend on the order of definition, but these should be rare enough and avoidable.

Reply:
Agreed, depending on the order is bad practice.
The order here is not practical, it this is done here for readability purposes only.

Reply:
I "readability" you refer here implies order, and we want to say there is no order. Or more accurately, we want to enforce order through the hierarchy of fixtures and not through the order they are referenced.
However, I am not going to argue about this further at this stage. Please try to avoid it next time.